locale/ja/manageiq.po

```diff
@@ -14136,7 +14136,7 @@ msgid ""
 "Datastore import was successful. Added/Updated %{namespace_stats} Namespaces, "
 "%{class_stats} Classes, %{instance_stats} Instances, %{method_stats} Methods."
 msgstr ""
-"データ・ストアが正常にインポートされました。 追加/更新された % {namespace_stats} の名前空間、%{class_stats} クラス、%"
+"データ・ストアが正常にインポートされました。 追加/更新された %{namespace_stats} の名前空間、%{class_stats} クラス、%"
```

Comment:
Oh it was _this_ one that was the interpolation bug.  I misunderstood and thought that the thing you commented on was the interpolation bug, which was why I was carrying on about the before/after